From c06b9cd500facfb6a10057490c1ec1090408ff12 Mon Sep 17 00:00:00 2001
From: Lijun Ou <oulijun@huawei.com>
Date: Wed, 6 Jan 2021 11:46:32 +0800
Subject: [PATCH 009/189] net/hns3: remove unnecessary memset

The hns3_cmd_desc has memset when setup and the memset
for req is unnecessary.

Signed-off-by: Lijun Ou <oulijun@huawei.com>
---
 drivers/net/hns3/hns3_rss.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/net/hns3/hns3_rss.c b/drivers/net/hns3/hns3_rss.c
index e2f0468..b5df374 100644
--- a/drivers/net/hns3/hns3_rss.c
+++ b/drivers/net/hns3/hns3_rss.c
@@ -633,16 +633,11 @@ hns3_set_rss_tc_mode(struct hns3_hw *hw)
 static void
 hns3_rss_tuple_uninit(struct hns3_hw *hw)
 {
-	struct hns3_rss_input_tuple_cmd *req;
 	struct hns3_cmd_desc desc;
 	int ret;
 
 	hns3_cmd_setup_basic_desc(&desc, HNS3_OPC_RSS_INPUT_TUPLE, false);
 
-	req = (struct hns3_rss_input_tuple_cmd *)desc.data;
-
-	memset(req, 0, sizeof(struct hns3_rss_tuple_cfg));
-
 	ret = hns3_cmd_send(hw, &desc, 1);
 	if (ret) {
 		hns3_err(hw, "RSS uninit tuple failed %d", ret);
-- 
2.7.4

